app/helpers/measures_helper.rb

```diff
@@ -33,4 +33,28 @@ def check_how_to_export_goods_link(declarable:, country_code:, country_name:, eu
       link_to('Check how to export commodity goods (link opens in new tab)', TradeTariffFrontend.check_duties_service_url, target: '_blank', rel: 'noopener')
     end
   end
+
+  def reformat_duty_expression(expression)
```

Comment:
Not sure this conveys the special snowflake purpose of this method. I'd probably add a comment saying it adjusts spans on component expressions that are conjoined to enable more sensible table wrapping